custom_components/ferroamp/sensor.py

```diff
@@ -1036,9 +1047,16 @@ def __init__(self, name, entity_prefix, key, icon, device_id, device_name, inter
             interval,
             precision,
             config_id,
+            state_class=SensorStateClass.TOTAL_INCREASING,
             **kwargs
         )
 
+    def add_event(self, event):
+        phases = self.get_phases(event)
+        if phases is not None and (phases["L1"] is not None or phases["L2"] is not None or phases["L3"] is not None):
+            if (phases["L1"] + phases["L2"] + phases["L3"]) > 0:
```

Comment:
There will still be a problem is only some of the phases have 0 values. Not sure if this happens though?

Reply:
Sounds really strange if that would happen and in that case I think this is the correct solution anyway.